EqualityComparer/TicketEqualityComparer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using AirportTicketBooking.Model.Classes;
+using System.Diagnostics.CodeAnalysis;
+
+namespace AirportTicketBooking.EqualityComparer;
+
+public class TicketEqualityComparer : EqualityComparer<Ticket>
+{
+    public override bool Equals(Ticket? x, Ticket? y)
+    {
+        if(x == null || y == null) return false;
+        if(x.Id == y.Id
```

Comment:
NIT: ```
return x.Id == y.Id
            && x.Time == y.Time
            && x.Flight.Class.Type == y.Flight.Class.Type;
```

Reply:
Changing it will affect the signeture of the overrided method.

Reply:
how?